src/pages/RecipesDetailedPage/RecipesDetailedPage.tsx

```diff
@@ -13,56 +15,20 @@ import DirectionsList from 'components/DirectionsList';
 import Text from 'components/Text';
 import Loader from 'components/Loader';
 
-type RecipeData = {
-    id: number;
-    preparationMinutes: string;
-    cookingMinutes: string;
-    image: string;
-    aggregateLikes: string;
-    readyMinutes: string;
-    servings: string;
-    title: string;
-    summary: any;
-    extendedIngredients: [];
-    equipment: [];
-}
 
 const RecipesDetailedPage: React.FC = () => {
-    const [recipe, setRecipe] = useState<RecipeData>();
-    const { id } = useParams();
-    const [isDetailedPageLoading, setIsDetailedPageLoading] = useState(true)
+    const recipeDetailedStore = useLocalStore(() => new RecipeDetailedStore)
+    const recipesStore = useLocalStore(() => new RecipesStore)
```

Comment:
1. тут точно нужен `RecipesStore`? обычно логика такая - под каждую страницу свой стор
2. забыл скобочки
```suggestion
    const recipeDetailedStore = useLocalStore(() => new RecipeDetailedStore())
    const recipesStore = useLocalStore(() => new RecipesStore())
```